apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationStatementForm.jsx

```diff
@@ -24,15 +22,8 @@ export const ModerationStatementForm = (props) => {
     }
   }
 
-  const removeAlert = (e) => {
-    this.setState({
-      alert: null
-    })
-  }
-
   return (
     <form className="general-form" onSubmit={handleSubmit}>
-      <Alert onClick={() => removeAlert()}>{translated.alertError}</Alert>
```

Comment:
sorry, not fimiliar with the code: we remove this because it was unused? the translation for `translated.alertError` can be removed then as well, right?

Reply:
oh yes..